lcls-twincat-pmps/PMPS/GVLs/PMPS_GVL.TcGVL

```diff
@@ -49,9 +49,10 @@ VAR_GLOBAL CONSTANT
 	
 	TRANS_SCALING_FACTOR : REAL := REAL#1.0; // Scaling factor for fixed-point transmission
 
-	
+	AUX_ATTENUATORS : UINT := 16; // Maximum # of attenuators in the PMPS
     MAX_VETO_DEVICES   :   UINT := 16;
-    {attribute 'pytmc' := '
+	stAttenuators : ST_PMPS_Attenuator :=(nTran:=1,xAttOK:=1);
```

Comment:
If we include `xAttOk:=1` in `cstFullBeam` and in `cst0RateBeam`, does that imply anything about the beam comparisons? `xAttOK` isn't used in `F_SafeBPCompare` right now but I wonder if it should be.

Reply:
If `xAttOK == 0`, can we even trust the `nTrans` value the attenuators report? Maybe I'm a bit confused as to what this indicates (or maybe we need to discuss this a bit more)...

Reply:
Currently, in the library it is not being used in the F_SafeBPCompare, or anywhere else, other than copying it from and to the _IO.
I think we need to define how to use it and where. We can also keep it just in the arbiter.